docs/deploy-development-rp.md

```diff
@@ -181,6 +213,58 @@ mock a cluster MSI. This script will also create the platform identities, platfo
 
    [1]: https://docs.microsoft.com/en-us/azure/openshift/tutorial-create-cluster
 
+### Create a MIWI cluster
+
+1. Ensure the required environment variables are set:
+
+   - `make aks.kubeconfig`
+   - `export ARO_INSTALL_VIA_HIVE=true` : instructs the RP to use hive to install clusters
+   - `export HIVE_KUBE_CONFIG_PATH=$PWD/aks.kubeconfig` : where to look for the kubeconfig
+
+1. Ensure that the required platform workload identities were created in your resource group. If they haven't been, run `./hack/devtools/msi.sh`
+
+   - aro-cloud-controller-manager
+   - aro-ingress
+   - aro-machine-api
+   - aro-disk-csi-driver
+   - aro-cloud-network-config
+   - aro-image-registry
+   - aro-file-csi-driver
+   - aro-aro-operator
+   - aro-Cluster
+
+1. Assign an additional role to your cluster managed identity
+
+   - `az role assignment create --role /subscriptions/fe16a035-e540-4ab7-80d9-373fa9a3d6ae/providers/Microsoft.Authorization/roleDefinitions/97c5b690-69b4-49a5-aa0a-3a685c28e3e6 --scope "/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$CLUSTER_RESOURCEGROUP" --assignee $MOCK_MSI_OBJECT_ID`
```

Comment:
Looks like we can pull this step out at this point. This was a workaround to assign a custom role that was put in place due to a missing permission in a built-in role, but since then the permission has been added and the `msi.sh` script should already take care of doing the correct role assignment.